### PR TITLE
Add privacy link to Get Firefox, update download link

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/addons-frontend/issues/5245

screenshot 2018-06-25 16 36 29

screenshot 2018-06-25 16 19 11
screenshot 2018-06-25 16 19 52
screenshot 2018-06-25 16 20 25

### CODE REVIEW

File: src/amo/components/Addon/styles.scss [hunk 1]
```diff
@@ -237,3 +242,22 @@
...
+  display: flex;
+  flex-direction: column;
+  width: max-content;
+
+  .Addon-get-firefox-privacy-link,
+  .Addon-get-firefox-privacy-link:active,
+  .Addon-get-firefox-privacy-link:link,
+  .Addon-get-firefox-privacy-link:visited,
+  .Addon-get-firefox-privacy-link:hover {
+    @include font-medium();
```
Comment (willdurand) [inline src/amo/components/Addon/styles.scss]: According to [this comment](https://github.com/mozilla/addons-frontend/issues/5245#issuecomment-395912680), the font size should `11px` and the font weight `700`. Currently we have `12px` and `500`. I find the current text too prominent, especially compared to this screenshot:



Last but not least, I'd expect some sort of `text-decoration` on hover/active.

Comment (kumar303) [reply src/amo/components/Addon/styles.scss]: Ah, thanks, I was looking at the wrong one in the spec.

**Comment by kumar303** [comment]
Thanks. It's ready for another review. I am a little hesitant to introduce custom styles for this one element so one thing I didn't change is the weight of our existing `font-bold()` which is pretty close at 600.

File: src/amo/components/Addon/styles.scss [hunk 2]
```diff
@@ -237,3 +242,26 @@
...
+  .Addon-get-firefox-privacy-link:visited,
+  .Addon-get-firefox-privacy-link:hover {
+    @include font-bold();
+
+    color: $ink-90;
+    font-size: $font-size-caption10;
+    text-decoration: none;
+  }
+
+  .Addon-get-firefox-privacy-link:hover {
```
Comment (willdurand) [inline src/amo/components/Addon/styles.scss]: should we also set this for `:focus` and maybe `:active`?

**Comment by kumar303** [comment]
> Do we really want to break the bottom-right positioning?

Whoops. That was a regression.

I also fixed the width of the button on mobile. It's ready for another look.

**Comment by willdurand** [comment]
Adding "state: do not merge" because of: https://github.com/mozilla/addons-frontend/issues/5245#issuecomment-400682971

**Comment by kumar303** [comment]
Closing for now

**Pull request closed without merging**